cosmovisor/README.md

```diff
@@ -1,4 +1,4 @@
-# Cosmosvisor
+# Cosmovisor
```

Comment:
probably want to add a note that if you are upgrading from 0.35- to 0.35+ you will need to upgrade cosmovisor.

Reply:
I think what you mean will be specified in this issue: https://github.com/cosmos/cosmos-sdk/issues/12039#issuecomment-1143638427